plugin/jetpack.vim

```diff
@@ -211,7 +211,13 @@ function! s:clean_plugins() abort
 endfunction
 
 function! s:make_download_cmd(pkg) abort
-  if g:jetpack_download_method ==# 'git'
+  if a:pkg.local
+    if has('win32')
+      return ['cmd.exe', '/c', join(['mklink', '/D', a:pkg.path, a:pkg.url])]
```

Comment:
A symbolic link is available for only system administrators.
Please consider using a hard link.

Reply:
or junction link. Please see https://so-zou.jp/software/tool/system/command-prompt/command/mklink.htm